source/administration/devops-command-center.rst

```diff
@@ -148,13 +155,12 @@ Join an incident by joining the incident channel. If the incident channel is pub
 
 If the incident channel is private, an existing member of the incident channel must invite you. System administrators may join private incident channels without permission.
 
-Working with stages and tasks
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Working with checklists and tasks
```

Comment:
Are we going to call them checklists? Or sections of one checklist? @itao ?

Reply:
Good question. We can change this to "Working with tasks" - it's also more scalable. Thoughts?

Reply:
@justinegeffen Sounds good!